modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc

```diff
@@ -4,42 +4,58 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../../assets/images/
 
-To configure a connector in Design Center, follow this process:
+Design Center enables you to create apps visually. To use Design Center, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
 
-. Configure the trigger for the connector, which can be an HTTP Listener, a Scheduler, or an operation from the connector (if available).
+To create an app in Design Center:
 
-. Configure the target component for the connector.
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
 
-== Configure the Amazon S3 Connector in Design Center
+For information about Amazon S3 fields, see the xref:amazon/amazon_S3_connector-reference[Amazon S3 Connector Reference].
 
-To configure the Amazon S3 connector in Design Center:
+== Configure the Trigger
```

Comment:
Please use "Configure the Input Source Trigger" -- we can't assume everyone gets the trigger concept.

Reply:
Is this the title we are using going forward?

Reply:
I will add this now. I think it needs to be discussed:

Configure the Input Source (Trigger)
Configure the Input Source Trigger